examples/async-examples.js

```diff
@@ -0,0 +1,59 @@
+(async () => {	
+	const revai = require('../../revai-node-sdk');
+
+	/* Initialize your client with your revai access token */
+	const token = '<YOUR-ACCESS-TOKEN>';
+	var client = new revai.RevAiApiClient(token);
+
+	/* Get account details */
+	var account = await client.getAccount();
+	console.log(`Account: ${account.email}`);
+	console.log(`Balance: ${account.balance_seconds} seconds`);
+
+	
```

Comment:
minor - extra space

Reply:
Do you have a linter to find this for you?

Reply:
We do have a linter. @menioa run `npm run lint`